packages/attention/stories/Attention.stories.tsx

```diff
@@ -213,5 +214,68 @@ export function Popover() {
         </ul>
       </Attention>
     </div>
+  </div>
+  )
+}
+
+export function PopoverIconAsTargetEl() {
+  const [show, setShow] = React.useState(false)
+  const containerRef = React.useRef<HTMLDivElement>(null)
+  const targetEl = React.useRef<any>(null);
+
+  React.useEffect(() => {
+    function onBlurHandler(e) {
+      if (containerRef.current && !containerRef.current.contains(e.target)) {
+        setShow(false)
+      }
+    }
+    document.addEventListener('mousedown', onBlurHandler)
+    return () => {
+      document.removeEventListener('mousedown', onBlurHandler)
+    }
+  })
+
+  return (
+  <div className='flex items-center justify-center'>
+    <div ref={containerRef}>
+      <Button
+        small
+        aria-expanded={show}
+        aria-controls='popover-icon-target-el-attention-example'
+        utility
+        quiet
+        onClick={() => setShow(!show)}
+        className='w-max mb-0'
+        ref={targetEl}
+      >
+        <IconInfo16 />
+      </Button>
+      <Attention
+        popover
+        placement='bottom-end'
+        flip
+        fallbackPlacements={['bottom']}
+        distance={2}
+        skidding={12}
+        targetEl={targetEl}
+        isShowing={show}
+      >
+        <ul className='bg-white w-full text-center'>
+          <li
+            tabIndex={0}
+            className='p-2 px-40 hover:bg-blue-200 hover:text-blue-500 cursor-pointer'
```

Comment:
`text-blue-500` and `bg-blue-200` won't work in Warp. It was probably copied from another example here that wasn't fixed after moving over from Fabric. These classes + `cursor-pointer` together with `tabIndex={0}` should probably be removed in order to show plain text, or you could add link elements as children to the `li` elements:
```
  <li className='p-2 px-40' >
    <a href='#/url1'>
      Link 1
    </a> 
  </li>
  <li className='p-2 px-40'>
    <a href='#/url2'>
      Link 2
    </a>
  </li>
```

Reply:
Nice catch! 😄 This has also been copied from the popover example. I have removed these classes + tabIndex from both popover and popover with icon as targetEl examples in this commit: [0985eb4](https://github.com/warp-ds/react/pull/215/commits/0985eb49b76424114e9d76f4cad8d3f695c8079b)